rotkehlchen/api/v1/schemas.py

```diff
@@ -2611,7 +2618,8 @@ def validate_schema(
         supported_blockchain_type = blockchain.get_chain_type()
         if ((
             supported_blockchain_type == 'bitcoin' and
-            is_valid_btc_address(data['address']) is False
+            is_valid_btc_address(data['address']) is False and
```

Comment:
shouldn't this better check the exact value and do `is_valid_btc_address` for `SupportedBlockchain.BITCOIN` and `is_valid_bitcoin_cash_address` for `SupportedBlockchain.BITCOIN_CASH` ?